mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule Samly.Mixfile do
     [
       {:plug, "~> 1.6"},
       {:esaml, "~> 4.3"},
-      {:sweet_xml, "~> 0.6"},
+      {:sweet_xml, ">= 0.0.0"},
```

Comment:
Currently, an update will allow for `0.x.y` versions, the proposed change will make `x.y.z` updates which I'd like to manage manually to prevent breaking changes from leaking through.